test/unit/logic/block.js

```diff
@@ -288,7 +288,7 @@ describe('block', function () {
 
 			var blockNormalizeStub;
 
-			before(function () {
+			beforeEach(function () {
```

Comment:
Now this is a `beforeEach`, the `after` hook is probably unnecessary. This also applies in a few other places.